pkg/chunkenc/memchunk.go

```diff
@@ -74,7 +88,7 @@ type MemChunk struct {
 	cutBlockSize int
 
 	// Current in-mem block being appended to.
-	head *headBlock
+	head HeadBlock
 
 	// the chunk format default to v2
```

Comment:
While we are at it, If I am not wrong, we default to `v3` now, and I think we should update this comment too?

Reply:
I think its not yet active since `NewMemChunk` still creates an ordered head.

Reply:
I found the [DefaultChunkFormat](https://github.com/grafana/loki/blob/23ad7f2e6178708523093d0c8293c5a7335dc5cb/pkg/chunkenc/memchunk.go#L35) being set to `v3`. Am I looking at the wrong place?

Reply:
No I missed it

Reply:
This is covered in the followup PR https://github.com/grafana/loki/pull/4001 :)